specs/sharding/p2p-interface.md

```diff
@@ -160,11 +150,6 @@ The following validations MUST pass before forwarding the `signed_blob_header` o
   Propagating nodes MAY increase fee increments in case of spam.
 - _[REJECT]_ The header signature, `signed_blob_header.signature`, is valid for ONLY the builder --
   i.e. `bls.Verify(builder_pubkey, blob_signing_root, signed_blob_header.signature)`. The signature is not an aggregate with the proposer.
-- _[REJECT]_ The header is designated for proposal by the expected `proposer_index` for the blob's `header.slot` and `header.shard`
```

Comment:
We still need to verify the shard proposer index (in other p2p topics too). Assuming we want the index to be embedded at all (I like it better, as the shuffling is not available the same way, and it allows for validating the signature ahead of the shuffling, just like with beacon blocks)

Reply:
OMG, deleted those even without reading. Sorry, fixing this...

Reply:
Fixed here: df01588